caravel/assets/visualizations/histogram.css

```diff
@@ -0,0 +1,24 @@
+.bars rect {}
+
+.axis path
+
+.axis line {
+  fill: none;
+  stroke: black;
+  shape-rendering: crispEdges;
+}
+
+.axis text {
+  font-family: sans-serif;
```

Comment:
let's use this for declaring font-family: 
`font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, sans-serif;`